attrs_strict/_error.py

```diff
@@ -89,3 +90,17 @@ def __str__(self):
         )
 
         return self._render(error)
+
+
+def _is_newtype(type_):
```

Comment:
Since this is not necessarily functionality related to errors, would you mind moving this into a new file, eg `_newtype.py` and add this there ? 

In that case you can import it in both `_error` and `_type_validation` and use it.

Reply:
Yes, but such a `_newtype.py` file would be very small, and it is unlikely that it will grow. Mhmm I named the file now `_commons.py` for things which are not directly related to
_error or _type_validation . With the name `_commons.py` you are able to add further things if it might be required in the future.